explorer/e2e/anvil/anvil-filters.spec.ts

```diff
@@ -0,0 +1,141 @@
+import { expect, test } from "@playwright/test";
+import {
+  filterRegex,
+  getFirstElementTextLocator,
+  testClearAll,
+  testFilterBubbles,
+  testFilterCounts,
+  testFilterPersistence,
+  testFilterPresence,
+} from "../testFunctions";
+import { anvilFilters, anvilTabs, anvilTabTestOrder } from "./anvil-tabs";
```

Comment:
This is outside the scope of this PR but would you be able to update `anvilFilters`, `anvilTabs`, and `anvilTabTestOrder` to be MACRO_CASE to help indicate they are constants?

Reply:
Yeah sure! I'll do it in the cleanup ticket